letter-sender/src/main/java/com/christmas/letter/sender/configuration/AwsConfig.java

```diff
@@ -0,0 +1,20 @@
+package com.christmas.letter.sender.configuration;
+
+import java.net.URI;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import software.amazon.awssdk.regions.Region;
+import software.amazon.awssdk.services.sns.SnsClient;
+
+@Configuration
+public class AwsConfig {
+
+  @Bean
+  public SnsClient configSns(){
+    return SnsClient.builder()
+        .region(Region.US_EAST_1)
+        .endpointOverride(URI.create("http://localhost:4566"))
+        .build();
```

Comment:
these properties usually change over time, they can be loaded from application.properties files

Reply:
I moved the endpoint in application.properties